innovation_sweet_spots/pipeline/pilot/timeseries_gtr.py

```diff
@@ -10,16 +10,18 @@
 
 # Parameters (tech categories to process, time series limits)
 PARAMS = import_config("iss_pilot.yaml")
-SPLIT = True
+SPLIT = False
+PERIOD = "month"
```

Comment:
I guess we should keep it to `"year"` as that's what we used to report figures in the pilot project:
```suggestion
PERIOD = "year"
```

Reply:
Or - alternatively - should provide a comment/instruction that this can be changed to year, month or quarter...